test/FunctionalTests/Microsoft.OData.Core.Tests/ContainerBuilderExtensionsTests.cs

```diff
@@ -15,44 +16,44 @@ public class ContainerBuilderExtensionsTests
         [Fact]
         public void AddServiceWithServiceType()
         {
-            IContainerBuilder builder = new TestContainerBuilder();
-            builder.AddService(ServiceLifetime.Transient, typeof(Foo));
+            var builder = new TestContainerBuilder();

```

Comment:
@KenitoInc  it's the `TestContainerBuilder` class. But I think we should get rid of it and just use `new ServiceCollection()` directly. `TestContainerBuilder` was needed cause we were mocking `IContainerBuilder`, but since we're getting rid of `IContainerBuilder` we can get rid of `TestContainerBuilder` as well.